app/assets/javascripts/views/project_velocity_view.js

```diff
@@ -5,7 +5,7 @@ const ProjectVelocityView = Backbone.View.extend({
 
   className: 'velocity',
 
-  initialize: function() {
+  initialize: function() {    
```

Comment:
```suggestion
  initialize: function() {
```
💅

Reply:
Seeing these comments, I think @felipebrunetti94 doesn't have editorconfig set up in his editor.

@felipebrunetti94 ping me when you get online so I can help you with that :smile: